editor/editor_node.cpp

```diff
@@ -60,6 +60,7 @@
 #include "editor/import/editor_import_collada.h"
 #include "editor/import/editor_scene_importer_gltf.h"
 #include "editor/import/resource_importer_bitmask.h"
+#include "editor/import/resource_importer_csv.h"
```

Comment:
File forgotten?

Reply:
@akien-mga Pardon? The `resource_importer_csv` is above while the `resource_importer_csv_translation` is below. Did I miss something?

Reply:
OH, geez. Yeah the importer isn't here. Whoops.